src/index.js

```diff
@@ -0,0 +1,114 @@
+import './style.css';
+
+class LeaderboardApp {
+  constructor() {
+    this.body = document.querySelector('body');
+    this.launcher();
+  }
+
+  launcher() {
+    const main = document.createElement('main');
+    main.classList.add('main');
+
+    const header = this.createHeader(); // Utilisation correcte de 'this'
+    const left = this.createLeftSection(); // Utilisation correcte de 'this'
+    const right = this.createRightSection(); // Utilisation correcte de 'this'
+
+    main.appendChild(header);
+    main.appendChild(left);
+    main.appendChild(right);
+
+    this.body.innerHTML = '';
+    this.body.appendChild(main);
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  createHeader() {
+    const header = document.createElement('div');
+    header.classList.add('header');
+    const h1 = document.createElement('h1');
+    h1.textContent = 'Leaderboard';
+    header.appendChild(h1);
+    return header;
+  }
+
+  createLeftSection() {
+    const left = document.createElement('div');
+    left.classList.add('left');
+
+    const div1 = this.createSectionWithTitle('Recent scores'); // Utilisation correcte de 'this'
+    const table = this.createScoreTable();
+    div1.appendChild(table);
+
+    left.appendChild(div1);
+    return left;
+  }
+
+  createRightSection() {
+    const right = document.createElement('div');
+    right.classList.add('right');
+
+    const div2 = this.createSectionWithTitle('Add your score'); // Utilisation correcte de 'this'
+    const form = this.createScoreForm();
+    div2.appendChild(form);
+
+    right.appendChild(div2);
+    return right;
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  createSectionWithTitle(titleText) {
+    const div = document.createElement('div');
+    div.classList.add('title');
+    const h2 = document.createElement('h2');
+    h2.textContent = titleText;
+    div.appendChild(h2);
+    return div;
+  }
+
+  createScoreTable() {
+    const table = document.createElement('table');
+    const tr = document.createElement('tr');
+    const td1 = this.createTableHeader('Libellé: 100'); // Utilisation correcte de 'this'
+    const td2 = this.createTableHeader('Libellé: 50'); // Utilisation correcte de 'this'
+    const td3 = this.createTableHeader('Libellé: 20'); // Utilisation correcte de 'this'
+    tr.appendChild(td1);
+    tr.appendChild(td2);
+    tr.appendChild(td3);
+    table.appendChild(tr);
+    return table;
+  }
```

Comment:
- Optional
I would recommend following the wireframe design. You can customize it however you want, but the basic layout should be the same. Also, the refresh button is missing in your case.

| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://github.com/GhostEsso/Leaderboard/assets/33268927/4f4054c0-864f-48c6-ab30-f0a9dc5e1a71) | ![image](https://github.com/GhostEsso/Leaderboard/assets/33268927/b4f7c1f2-f84c-4135-b106-ff45f7aa1a28) |

Reply:
@muneebulrehman Okay, I'm going to change it :pray:

Reply:
@muneebulrehman  I did it : 
![image](https://github.com/GhostEsso/Leaderboard/assets/114746721/00e14e66-b772-4446-8d62-fbb524e8327f)